include/fc/log/zipkin.hpp

```diff
@@ -45,6 +45,9 @@ class zipkin_config {
    /// Starts with a random id and increments on each call, will not return 0
    static uint64_t get_next_unique_id();
 
+   /// Handle SIGHUP signal forwarded from net_plugin
```

Comment:
`fc` is a general purpose library. This is not necessarily only called from net_plugin. net_plugin is not part of `fc`. Also your implementation assumes this is not called directly from a SIGHUP signal handler. It would be good to note that here. Our use of signals in appbase makes this call from the main thread, but this should not be assumed. Either make this method signal safe (https://thomastrapp.com/blog/signal-handler-for-multithreaded-c++) OR clearly document how this can be called.

Reply:
Made this method signal safe.